packages/react/src/primitives/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import {
+  BaseBreadcrumbProps,
+  BreadcrumbProps,
+  Primitive,
+  ForwardRefPrimitive,
+} from '../types';
+import { ComponentClassNames } from '../shared/constants';
+import { View } from '../View';
+import { BreadcrumbItem } from './BreadcrumbItem';
+import { BreadcrumbLink } from './BreadcrumbLink';
+import { BreadcrumbsContext } from './BreadcrumbsContext';
+
+const BreadcrumbsPrimitive: Primitive<BreadcrumbProps, 'nav'> = (
+  { className, children, separator = '/', ...rest },
+  ref
+) => {
+  const componentClasses = classNames(
+    ComponentClassNames.Breadcrumbs,
+    className
+  );
+  const value = React.useMemo(() => {
```

Comment:
eslint was telling me I need to memoize this or it'll cause a re-render every time. Seems silly though?

Reply:
I think that's because you're wrapping `separator` in an object, which will be a new object every time. If `value` is a new object on every render, then it will cause renders to all component consuming the context. If you don't want to do this, you could just pass separator straight to value since that appears to be the only thing in your context value.